instrumentation/http/src/main/java/brave/http/aws/XRayHttpServerParser.java

```diff
@@ -0,0 +1,27 @@
+package brave.http.aws;
+
+import brave.SpanCustomizer;
+import brave.http.HttpAdapter;
+import brave.http.HttpServerParser;
+import zipkin.TraceKeys;
+
+public class XRayHttpServerParser extends HttpServerParser {
```

Comment:
I think we'll end up with something like XRayParser.httpServer() httpClient() sqlClient() etc. If we do our dependencies well, then we can have brave-parser-xray live in zipkin-aws and only depend on brave-instrumentation-http and a future brave-instrumentation-sql

Reply:
@adriancole Can you expand this a little bid more? Since TracingStatementInterceptor are appended to sql connection strings, it made me confused.

Reply:
I mean to create a generic parser for SQL databases, similar to what we have in brave-instrumentation-http. ex a new support module which would allow you to plug-in SqlClientParser like HttpClientParser. ex brave-instrumentation-sql

Reply:
if you did this, it would decouple xray from each individual sql instrumentation library